.github/workflows/checksums.yml

```diff
@@ -0,0 +1,27 @@
+name: checksums
+
+# Controls when the workflow will run
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  generate-checksums:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create checksums
+        run: |
+          git config user.name 'GitHub Actions'
```

Comment:
Are those username and email valid/work with git (have access to the)?

Reply:
I believe so. But I think we should put the user name/email of a user with write access.

Reply:
@Itai-Nelken I've put Botspot as the committer. It should work now.

Reply:
@Botspot is this fine with you?